error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/BigDecimalRulesTestOutput.java

```diff
@@ -31,25 +31,25 @@ ImmutableSet<Boolean> testBigDecimalSignumIsZero() {
 
   ImmutableSet<Boolean> testBigDecimalSignumIsPositive() {
     return ImmutableSet.of(
-        BigDecimal.valueOf(1).signum() > 0,
-        BigDecimal.valueOf(2).signum() > 0,
-        BigDecimal.valueOf(3).signum() > 0,
-        BigDecimal.valueOf(4).signum() > 0,
-        BigDecimal.valueOf(5).signum() <= 0,
-        BigDecimal.valueOf(6).signum() <= 0,
-        BigDecimal.valueOf(7).signum() <= 0,
-        BigDecimal.valueOf(8).signum() <= 0);
+        BigDecimal.valueOf(1).signum() == 1,
+        BigDecimal.valueOf(2).signum() == 1,
+        BigDecimal.valueOf(3).signum() == 1,
+        BigDecimal.valueOf(4).signum() == 1,
+        BigDecimal.valueOf(5).signum() != 1,
```

Comment:
if I understand correctly how this works, IMO the `AlsoNegation` now doesn't make sense anymore (or will these `!= 1`s be rewritten in a second pass?)

Reply:
Nope, `!= 1` means "the value is non-positive", while `!= -1` means "the value is nonnegative" (i.e., what was earlier expressed as `<= 0` and `>= 0`, respectively).

(In my most recent commit I only swapped the "canonical presentation"; there's no semantic change. Good to keep in mind that for the second half of the test cases, the test method is misleading (which is unavailable due to our test framework).)

Reply:
Ah right 👍🏻